src/components/TopBar.js

```diff
@@ -5,17 +5,20 @@ import { AppBar, Toolbar, Button, Typography } from '@material-ui/core';
 const useStyles = makeStyles((theme) => ({
   root: {
     flexGrow: 1,
-    color: theme.palette.text.primary,
-    background: theme.palette.warning.dark,
-    position: 'static'
+    color: 'white',
+    background: '#30638E',
+    position: 'static',
+    height: '50%'
   },
   docButton: {
-    margin: theme.spacing(2)
+    margin: theme.spacing(1),
+    color: 'white',
+    background: 'linear-gradient(45deg, #30638E 30%, #285a85 90%)',
+    boxShadow: '0 3px 5px 2px #285a85'
   },
   title: {
     flexGrow: 1,
-    edge: 'start',
-    fontFamily: 'Consolas'
```

Comment:
Did you remove the `fontFamily` here as part of matching the top bar with FSH School? I'm just curious, especially since we were talking about the different fonts earlier.

Reply:
Yes, I was trying to get it to most closely resemble the font used in the fsh-school or fsh-online.

Reply:
Ah cool! Makes sense to me!